_pages/about-us/teams/OA-operational-guidance/management-reviews-of-procurement-actions.md

```diff
@@ -38,13 +40,13 @@ STANDARD OPERATING PROCEDURE: TC-18-CS1-001
       - Award decision memo
 
   - **For procurements above $25,000:**
-    - TTS management (one level above the CO) and TTS procurement analysts will perform the first review of acquisition packages. The Office of Governmentwide Policy (OGP) will conduct additional reviews, per OGP’s request and the business process as it stands at the issuing of this policy.
-    - At the request of OGP, reviews may be conducted prior to solicitation/RFQ release and/or prior to award or any other stage in the procurement process.
+    - TTS management (one level above the CO) and TTS procurement analysts will perform the first review of acquisition packages prior to RFQ/solicitation release and prior to award. 
+    - The Office of Governmentwide Policy (OGP) may conduct additional reviews, per OGP’s request at any stage in the procurement process.
     - See above for examples of acquisition package documents to be reviewed.
```

Comment:
OGP may conduct additional reviews per OGP's request?

Reply:
Yes, they may request to review certain procurement packages.

Reply:
I think the wording is just kind of odd. They can do reviews based on their own request? Wouldn't that just be at their discretion or something like that?

"Baxter may conduct additional code reviews at Baxter's request." -- see what I mean?

Reply:
Sorry your comment wasn't clear. I thought you were seeking clarification rather than editing the wording. I understand your comment now.